src/main/java/file/FileRenameSnippet.java

```diff
@@ -0,0 +1,44 @@
+package file;
```

Comment:
1. Check the other files and update the file header please. 
2. Also squash all the commits.
3. Just add 1 empty line at the EOF instead of adding multiple lines. Also no need to write anything in the empty line, it will defeat the purpose of empty line addition. I can see you have written "//EOF" which is flawed.

With these changes, the PR looks good to me.